### PR TITLE
make signup payment text minimum fifteen pounds

### PR DESCRIPTION
**Pull request by CllrScruffles**
The board has decided to raise the minimum amount we ask for new sign ups to pay £15. Of course, people are still able to input a lower amount (even lower than £12.50) but this requires a longer discussion around code changes. I have however amended the text in the sign up to reflect that we are asking people this, so as to avoid having people join now and then ask them in 2, 3 (or n months) to raise it to £15. AFAIK emails have been sent to members paying below £15 (and who are active in the space) about raising their subscription.

### CODE REVIEW

File: resources/views/account/create.blade.php [hunk 1]
```diff
@@ -165,7 +165,7 @@
         @else
         <ul>
             <li>£20 a month, like a cheap gym membership, is the recommended amount for new starters.</li>
-            <li>You can pay less, perhaps if you're on lower income.</li>
+            <li>You can pay less, perhaps if you're on lower income, but we do ask for a minimum of £15. Click the link above for more advice.</li>
```
Comment (chris18890) [inline resources/views/account/create.blade.php:168]: should this be &pound; or £?

Comment (CllrScruffles) [reply resources/views/account/create.blade.php:168]: Lol yes, sorry

File: resources/views/account/create.blade.php [hunk 2]
```diff
@@ -290,7 +290,7 @@
                 </p>
 
                 <p>
-                    If you would like to pay less than &pound;12.50 a month please select an amount over £12.50 and complete
+                    If you would like to pay less than &pound;15 a month please select an amount over £15 and complete
```
Comment (chris18890) [inline resources/views/account/create.blade.php:293]: should this be `&pound;` or `£` in both places?

**Review by chris18890 — CHANGES_REQUESTED** [review]
Suggest you change the `£` to `&pound;` for consistency

**Comment by conorriches** [comment]
Looks good, it would be worth also amending `app/Http/Controllers/AccountController.php` lines 482 and 484, so that people can't then lower their amount under £15.

**Review by chris18890 — APPROVED** [review]

**Comment by CllrScruffles** [comment]
Have amended the validation check in AccountController.php

**Comment by conorriches** [comment]
Didn't get there in time, was just finding the file but I'll update the UserValidator too, so that users can't edit the form and submit a value under 15

**Comment by CllrScruffles** [comment]
> Didn't get there in time, was just finding the file but I'll update the UserValidator too, so that users can't edit the form and submit a value under 15

nice one cheers

**Pull request merged**